frameworks/autoxgboost/exec.R

```diff
@@ -0,0 +1,31 @@
+library(mlr)
+library(autoxgboost)
+library(farff)
+
+run = function(train_file, test_file, output_predictions_file, cores, time.budget) {
+  train = farff::readARFF(train_file)
+  colnames(train) = make.names(colnames(train))
+  target = colnames(train)[ncol(train)]
```

Comment:
as explained in https://github.com/openml/automlbenchmark/pull/50#issuecomment-527243835
you need to find a way to use `dataset.target.index` here instead of `ncol(train)`.

Reply:
@Coorsaa any update on this?